java/code/src/com/redhat/rhn/manager/formula/FormulaManager.java

```diff
@@ -406,11 +406,14 @@ private FormulaData getCombinedFormulaDataForSystemAndFormula(MinionServer minio
      * @param systemIDs list of system IDs
      * @return list of endpoint information objects
      */
-    public List<EndpointInfo> listEndpoints(List<Long> systemIDs) {
-        List<MinionServer> minions = MinionServerFactory.findMinionsByServerIds(systemIDs);
+    public List<EndpointInfo> listEndpoints(List<Number> systemIDs) {
```

Comment:
To me it would feel more sensible to keep `List<Long>` as the parameter here, because `Long` is the type we use everywhere in the business logic layer for the server ids. Since this is basically an XML-RPC issue, I think it's better to handle the discrepancy by doing this conversion in the handler itself.

Reply:
Agree. Done.